README.md

```diff
@@ -5,8 +5,17 @@
 This repository is the mobile application of Galoy. It is build with [React Native](https://reactnative.dev/), and compatible on iOS and Android.
 
 ## Start
+Prerequisite -- Follow the instruction [getting started](https://reactnative.dev/docs/getting-started) to have the correct setup.
 
-type `react-native run-ios` or `react-native run-android` to run locally. Follow the instruction [getting started](https://reactnative.dev/docs/getting-started) to have the correct setup.
+cd into project folder
+
+type `npm install`
```

Comment:
We use yarn internally for the project, so I think we should recommend `yarn install` instead. Or have both: `yarn install` or `npm install` .